### PR TITLE
Resolving issues - forester version 1.1.3

### PR DESCRIPTION
**Pull request by HubertR21**
Issues: [51](https://github.com/ModelOriented/forester/issues/51) [53](https://github.com/ModelOriented/forester/issues/53) [54](https://github.com/ModelOriented/forester/issues/54) [55](https://github.com/ModelOriented/forester/issues/55) [57](https://github.com/ModelOriented/forester/issues/57) [60](https://github.com/ModelOriented/forester/issues/60) [61](https://github.com/ModelOriented/forester/issues/61) [63](https://github.com/ModelOriented/forester/issues/63) [64](https://github.com/ModelOriented/forester/issues/64) [81](https://github.com/ModelOriented/forester/issues/81) [83](https://github.com/ModelOriented/forester/issues/83)

### CODE REVIEW

**Review by grudzienAda — APPROVED** [review]
Drobna stylistyka.

**Review by kozaka93 — APPROVED** [review]

File: R/check_data.R [hunk 1]
```diff
@@ -300,30 +309,50 @@ check_cor <- function(df, y, verbose = TRUE) {
 
     for (i in 1:length(fct_idx)) {
       for (j in 1:length(fct_idx)) {
-        cor_fct[i, j] <- round(rcompanion::cramerV(fct_tbl[, i], fct_tbl[, j]), 2)
+
+        a = as.numeric(length(unique(fct_tbl[, i])))
+        b = as.numeric(length(unique(fct_tbl[, j])))
+
+        if (as.numeric(a * b) <= 2^29 - 1){ # with 2^31 we have to big vector that can't be allocated (over 7.4 Gb)
```
[the remark this answers is not in the document]
Comment (grudzienAda) [reply R/check_data.R:316]: ?

File: R/train.R [hunk 2]
```diff
@@ -19,22 +19,10 @@
...
-#' one out of `default`/`portfolio`/`bayesian`/`random search`.
-#' If `default` then no tuning, just default hyperparameteres are used.
-#' If `portfolio` then static portfolio of hyperparametes are tried.
-#' If `bayesian` then Bayesian optimization is performed.
-#' If `random search` then random search is performed.
-#' @param keep Shall all models be returned (`keep = FALSE`) or only the best one
-#' (`keep = FALSE`, default option)?
 #' @param verbose A logical value, if set to TRUE, provides all information about
 #' training process, if FALSE gives none.
+#' @param train_test_split A 3 value vector, describing the proportions of train,
```
Comment (grudzienAda) [inline R/train.R:24]: 3-value?

File: R/train.R [hunk 3]
```diff
@@ -19,22 +19,10 @@
...
+#' test, validation subsets to original data set. Default values are: c(0.6, 0.2, 0.2).
 #' @param bayes_iter An integer value describing number of optimization rounds
 #' used by Bayesian optimization.
```
Comment (grudzienAda) [inline R/train.R:27]: by the

**Review by grudzienAda — APPROVED** [review]

**Pull request merged**